db/sqlinterfaces.c

```diff
@@ -9378,10 +9378,11 @@ int emit_sql_row(struct sqlthdstate *thd, struct column_info *cols,
         cols = malloc(ncols * sizeof(struct column_info));
         for (col = 0; col < ncols; col++) {
             const char *ctype = sqlite3_column_decltype(stmt, col);
+            const size_t size = sizeof(cols[col].column_name);
             cols[col].type = typestr_to_type(ctype);
-            snprintf(cols[col].column_name, sizeof(cols[col]), "%s",
+            snprintf(cols[col].column_name, size, "%s",
                      sqlite3_column_name(stmt, col));
-            cols[col].column_name[sizeof(cols[col]) - 1] = 0;
+            cols[col].column_name[size - 1] = 0;
```

Comment:
We don't really need this as snprintf() will take care of null-terminating the string. I'd, thus, use ```sizeof(cols[col].column_name)``` directly in snprintf().

Reply:
@nirbhayc done 😃